packages/marshal/src/marshal.js

```diff
@@ -255,7 +257,7 @@ export const makeMarshal = (
 
     /**
      * @param {{errorId?: string, message: string, name: string}} errData
-     * @param {(e: unknown) => Passable} decodeRecur
+     * @param {(e: unknown) => string} decodeRecur
```

Comment:
`decodeRecur` is the general decoder function and is not limited to returning strings.